configs/components/fesom/Subjob_tripy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#Home directory and reset of variable
+cd ~
+source ~/.bash_profile
+
+cd ~
+
+# Function to check for py39 envirionmet. If necessary installing and activating.
+#this is just to not trigger errors while activating py39
+#module unload netcdf-c
+#module unload netcdf-fortran
+#Function for activating py39
+activate_env() {
+    cd ~
+    source ~/.bash_profile
+    conda activate py39
```

Comment:
I think a better name than `py39` can be used, something that is more specific to this job, maybe `esm-tools_post_tripyview` (in general: `esm-tools_[job_part]_[specifier]`)